TuneUpTests/TuneUpTests.csproj

```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>TuneUpTests</RootNamespace>
     <AssemblyName>TuneUpTests</AssemblyName>
-    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7</TargetFrameworkVersion>
```

Comment:
any reason you decided to downgrade? I actually think 4.7.2 comes with 2019 - where as .net 4.7 is required to be installed separate.

Reply:
@mjkkirschner Oh, I only downgrade because Tuneup itself is still using 4.7, as well as Dynamo. No specific reason, I just thought at least within the repo we want to be consistent